src/cluster/src/cli/start/sys.rs

```diff
@@ -0,0 +1,40 @@
+use crate::cli::start::StartOpt;
+use crate::cli::ClusterCliError;
+use crate::sys::{SysConfig, SysInstaller};
+use crate::ClusterError;
+use crate::error::SysInstallError;
+
+pub fn install_sys(opt: StartOpt, upgrade: bool) -> Result<(), ClusterCliError> {
+    install_sys_impl(opt, upgrade).map_err(ClusterError::InstallSys)?;
+    Ok(())
+}
+
+fn install_sys_impl(opt: StartOpt, upgrade: bool) -> Result<(), SysInstallError> {
+    let mut builder = SysConfig::builder();
+    builder.with_namespace(opt.k8_config.namespace);
```

Comment:
Can collapse into a single line?

Reply:
Ooh, with the current code layout that was not possible, but I just thought of something that will make this work